x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/header/index.test.tsx

```diff
@@ -180,4 +196,32 @@ describe('Actions', () => {
 
     expect(generate).toBeDisabled();
   });
+
+  it('invokes openFlyout when the settings button is clicked, when the attackDiscoveryAlertFiltering feature flag is on', async () => {
+    (defaultAssistantFeatures.attackDiscoveryAlertFiltering as jest.Mocked<boolean>) = true;
+    const openFlyout = jest.fn();
+
+    render(
+      <TestProviders>
+        <Header
+          stats={null}
```

Comment:
nit: consider extracting props into an object and spreading so it is easy to identify which properties are being changed/tested